README.md

```diff
@@ -177,13 +178,13 @@ eclair-node-<version>-<commit_id>/bin/eclair-node.sh -Dlogback.configurationFile
 ### Backup
 
 You need to backup:
-- your bitcoin core wallet
-- your eclair channels
+- your Bitcoin core wallet
```

Comment:
nit: maybe also capitalize "Core"? Same in a few other places

```suggestion
- your Bitcoin Core wallet
```

Reply:
Good point; I'll go through and make sure Core is consistently capitalized.

Reply:
Seems this was the only place we needed to s/core/Core

Reply:
Are you sure? I see another one two lines below ;)

Reply:
Indeed! ha, not sure how I missed that. Anyway, fixed last commit.